shared/draft-utils/process-message-content.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import { stateFromMarkdown } from 'draft-js-import-markdown';
+import { convertFromRaw, convertToRaw, EditorState } from 'draft-js';
+import { addEmbedsToEditorState } from './add-embeds-to-draft-js';
+
+export default (type: 'TEXT' | 'DRAFTJS', body: string): string => {
```

Comment:
The `type` argument here should be lower case to match the input `message.messageType`. 

That way, you do not have to have a conditional outside of this method and can simply do

```JS
message.content.body = processMessageContent(message.messageType, message.content.body);
message.messageType = 'draftjs'
```

Reply:
To be honest, I just copied most of this file from here: https://github.com/withspectrum/spectrum/blob/alpha/shared/draft-utils/process-thread-content.js#L6 and removed one property that was irrelevant for messages. Could have done this refactor better probably, but I just didn't want to change the already existing naming of the stuff. If I change this here, it should be changed everywhere else too. 

The `type` argument / property for thread-related stuff is uppercase in the code. Is it lowercase like this anywhere else? Otherwise I'd suggest just uppercasing it here for the sake of consistency.

Reply:
It is inconsistent in the database, but it should be consistent with that. Threads have a `type` property that is either `TEXT` or `DRAFTJS`, but messages have a `messageType` property that is either `text` or `draftjs`.

That is inconsistent with each other, which is bad, but this method shiuld be consistent with the database, if that makes sense?

Reply:
Oh, I was not aware of that, I guess. Will change it then.

Reply:
Yep I figured, that's why I left that review comment. Sorry for not giving the full context in the initial comment!

Reply:
Okay, I committed some changes to fix it. While on it, I also noticed that those strings (`'text'` and `'draftjs'`) and so on, are all over the code, so I made some sort of "enum" for it and replaced all occurencies I could find using that. Strings are prone to typos, this should avoid it for future use. I can revert that and just fix the above, if that's required, but I think it's a good addition.